frontend/components/draft/lesson/lesson-renderer.tsx

```
@@ -64,6 +71,25 @@ export function LessonRenderer({ lesson, dropletSlug }: LessonRendererProps) {
     [lesson.id],
   );
 
+  const updateBlocksV2Backend = useCallback(
+    async (blocksV2: any) => {
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)